addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -381,6 +381,11 @@ def test_timesheet_invoice(self):
             'product_uom_qty': 20,
             'order_id': sale_order.id,
         })
+        so_line_deliver_timesheet = self.env['sale.order.line'].create({
+            'product_id': self.product_delivery_timesheet1.id,
+            'product_uom_qty': 5,
+            'order_id': sale_order.id,
+        })
```

Comment:
We need to create a new SOL to correctly reproduce the issue because `so_line_deliver_task_project` contains a project/task on its recordset and so the project manager can read the SOL thanks to the ir.rule added in `sale_project` module.